custom/hmhb/management/commands/repeat_forms_by_external_id.py

```diff
@@ -0,0 +1,51 @@
+"""
+To run this command for a CSV file of external IDs (e.g. tracked entity
+instance UIDs):
+
+Get the external IDs (assuming they are in the first column)::
+
+    $ cat my_csv_file.csv | cut -d, -f1
+
+Iterate them, and run this command for each one::
+
+    $ for extid in `cat my_csv_file.csv | cut -d, -f1`
+      do
+          ./manage.py repeat_forms_by_external_id \
+              my-domain \
+              abc123def456 \
+              $extid \
+              my-case-type
+      done
+
+"""
+from django.core.management.base import BaseCommand
+
+from corehq.form_processor.models import CommCareCase, XFormInstance
+from corehq.motech.repeaters.models import Repeater
+
+
+class Command(BaseCommand):
+    help = ('Finds the case identified by `external_id`, and triggers '
+            'a repeater for each form that updated that case.')
+
+    def add_arguments(self, parser):
+        parser.add_argument('domain')
+        parser.add_argument('repeater_id')
+        parser.add_argument('external_id')
+        parser.add_argument('case_type')
+
+    def handle(self, *args, **options):
+        repeater = Repeater.get(options['repeater_id'])
```

Comment:
Should this use `SQLRepeater` when https://github.com/dimagi/commcare-hq/pull/32160 is merged? (I think that will happen soon.)

Reply:
Yes. Thank you for flagging.